src/app/files.js

```diff
@@ -107,6 +107,47 @@ function Files (storage) {
     return files
   }
 
+  //
+  // Tree model for files
+  // {
+  //   'a': { }, // empty directory 'a'
+  //   'b': {
+  //     'c': {}, // empty directory 'b/c'
+  //     'd': { '/readonly': true, '/content': 'Hello World' } // files 'b/c/d'
+  //     'e': { '/readonly': false, '/path': 'b/c/d' } // symlink to 'b/c/d'
```

Comment:
it's just that i felt `/readonly` and `/mode` feel arbitrary. I'm used to `fs.stats(...)` so it feels "natural" and an empty space as key is invalid as a filename too...

in case of `b`, the "content" would directly be `fs.b`, but in case of `b/c`, the content would be `fs.b['/content'].c` ... if c is a file, but `fs.b.c` if c is a directory.

Where would you store "meta information" about directories?